### ISSUE TITLE
DropDownList does not open in Upload FileTemplateDirective

### ISSUE THREAD

**Issue by dcsweet** [telerik/kendo-angular#840]
## I'm submitting a...

* Bug report 

## Current behavior
DropDownList is immediately closed after opening when placed inside a FileTemplateDirective of the Upload component.

## Expected behavior
DropDownList should open to display its contents and remain in focus.

## Minimal reproduction of the problem with instructions

See plunk: http://plnkr.co/edit/ipr7F7IFWGaPCBtQWWCq?p=preview

1. Open plunk.
2. Upload a file.
3. Click the DropDownList and observe the behavior.

A second DropDownList is used in the plunk outside of the Upload component for reference and demonstration of proper functionality.

## What is the motivation or use case for changing the behavior?
In order associate file specific metadata from user input, the user must have usable controls rendered for each file in the list. A dropdown list is an ideal component for providing a limited selection of metadata options.

## Environment

Package versions:

```
+-- @angular/animations@4.3.0
+-- @angular/cli@1.2.0
+-- @angular/common@4.3.0
+-- @angular/compiler@4.3.0
+-- @angular/compiler-cli@4.3.0
+-- @angular/core@4.3.0
+-- @angular/forms@4.3.0
+-- @angular/http@4.3.0
+-- @angular/language-service@4.3.0
+-- @angular/platform-browser@4.3.0
+-- @angular/platform-browser-dynamic@4.3.0
+-- @angular/router@4.3.0
+-- @progress/kendo-angular-buttons@1.0.5
+-- @progress/kendo-angular-dateinputs@1.0.5
+-- @progress/kendo-angular-dialog@1.0.4
+-- @progress/kendo-angular-dropdowns@1.1.1
+-- @progress/kendo-angular-excel-export@1.0.3
+-- @progress/kendo-angular-grid@1.2.1
+-- @progress/kendo-angular-inputs@1.0.6
+-- @progress/kendo-angular-intl@1.2.1
+-- @progress/kendo-angular-l10n@1.0.2
+-- @progress/kendo-angular-popup@1.1.1
+-- @progress/kendo-angular-upload@1.1.1
+-- @progress/kendo-data-query@1.0.5
+-- @progress/kendo-drawing@1.1.1
+-- @progress/kendo-theme-default@2.38.0
+-- @types/jasmine@2.5.53
+-- @types/jasminewd2@2.0.2
+-- @types/node@6.0.83
+-- codelyzer@3.0.1
+-- core-js@2.4.1
+-- jasmine-core@2.6.4
+-- jasmine-spec-reporter@4.1.1
+-- karma@1.7.0
+-- karma-chrome-launcher@2.1.1
+-- karma-cli@1.0.1
+-- karma-coverage-istanbul-reporter@1.3.0
+-- karma-jasmine@1.1.0
+-- karma-jasmine-html-reporter@0.2.2
+-- office-ui-fabric-core@7.2.0
+-- protractor@5.1.2
+-- rxjs@5.4.2
+-- ts-node@3.0.6
+-- tslint@5.3.2
+-- typescript@2.3.4
`-- zone.js@0.8.13
```

Browser:

- Chrome (desktop) version 60.0.3112.90 (Official Build) (64-bit)

System:
angular/cli: 1.2.6
node: 7.10.0
os: win32 x64

**Comment by siyka-marinova** [comment]
Fix for this issue is currently available in the latest dev. version of the kendo-angular-upload package: `@progress/kendo-angular-upload@1.2.1-dev.201708300744`. The official package will be release shortly

**Comment by Raisolution** [comment]
Fixed in v1.2.1.
